src/util/config/build.js

```diff
@@ -84,23 +28,6 @@ export const schema = {
 };
 export const config = convict(schema);
 
-// Load environment dependent configuration
-const configPath = path.resolve(
```

Comment:
We don't need a local config for `env, `isProd`, `isDev`, but removing this means that we can't change the value of `disable_hmr` with a local config file. Shouldn't we be able to?